includes/Core/Modules/Modules.php

```diff
@@ -1119,4 +1127,45 @@ function( Module $module ) {
 		);
 	}
 
+	/**
+	 * Gets the recoverable modules.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array List of recoverable modules.
+	 */
+	public function get_recoverable_modules() {
+		$recoverable_modules = array();
+		$shareable_modules   = $this->get_shareable_modules();
+
+		foreach ( $shareable_modules as $module ) {
+			$owner_id = $module instanceof Module_With_Owner && $module->get_owner_id();
+
+			// If no owner identified by its owner_id OR lacks the AUTHENTICATE capability -
+			// push the module slug to the recoverableModules array.
+			if ( false === $owner_id || ! user_can( $owner_id, Permissions::AUTHENTICATE ) ) {
```

Comment:
The `get_owner_id()` method can return `0` which also means that there is no owner assigned.
```suggestion
			if ( empty( $owner_id ) || ! user_can( $owner_id, Permissions::AUTHENTICATE ) ) {
```